### DISCUSSION TITLE
How can I use slidev on a thumb drive like a portable app?

### DISCUSSION THREAD

**Discussion by kookma** [slidevjs/slidev#649] [Q&A]
Using slidev for meeting and presentation, one may need to run the slidev on a third party machine.
How can I have slidev as a portable app everywhere?

**Comment by michael-azogu** [comment]
Slidev is built on vite & vue (web & nodejs) it needs to be run in a node environment it's not a desktop app. 👍

**Comment by michael-azogu** [comment]
you can build the project & export slides as pdf or deploy to the web to be viewed publicly if you want to.
